components/cam/src/physics/cam/micro_mg2_0.F90

```diff
@@ -2754,6 +2757,11 @@ subroutine micro_mg_tend ( &
         if (dumc(i,k).ge.qsmall) then
 
 
+           ! impose minimum droplet number conc (in-cloud) 
+           if (mincdnc.gt.0._r8) then
+              nctend(i,k)=( max(nc(i,k)+nctend(i,k)*deltat, mincdnc/rho(i,k)) - nc(i,k))/deltat
```

Comment:
This nctend seems to be for whole grid (instead of in-cloud). should the limitation term be multiplied by lcldm?